examples/prometheus/prometheus.yaml

```diff
@@ -99,7 +106,8 @@ objects:
       spec:
         serviceAccountName: prometheus
         containers:
-        - name: oauth-proxy
+        # Deploy Prometheus behind an oauth proxy
+        - name: prometheus-oauth-proxy
```

Comment:
Call this prom-proxy since we're going to be typing it all the time, and the other container "prom", and then "alert-proxy" and "alert".